docs/manage-tags-in-gravitino.md

```diff
@@ -0,0 +1,328 @@
+---
+title: "Manage tags in Gravitino"
+slug: /manage-tags-in-gravitino
+date: 2024-07-24
+keyword: tag management, tag, tags, Gravitino
+license: This software is licensed under the Apache License version 2.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Introduction
+
+Starting from 0.6.0, Gravitino introduces a new tag system that allows you to manage tags for
+metadata objects. Tags are a way to categorize and organize metadata objects in Gravitino.
```

Comment:
What's the definition of `metadata objects`, does it the same as the entities in Gravitino? If it's a new concept or definition, please describe it.